app.js

```diff
@@ -2,29 +2,24 @@
 import express from "express";
 //import cors
 import cors from "cors";
-
 // import database
 import db from "./config/db.config.js";
-
 //import router
 import Router from "./routes/routes.js";
-await db.sync({ alter: true });
+//await db.sync({ alter: true });
 
 // models
 import "./models/index.js";
-
 const app = express();
 app.use(express.json());
-
+app.use(express.static("public"));
```

Comment:
@mukeshsin what is the use of this line? Can you explain?

Reply:
express. static ko ek  middleware function h ke roop mai use kr skte hain  express static mean serving static file in express such as image ,css file ,js file or html file bhi .

Reply:
@mukeshsin what is the use of this in your project?

Reply:
serving uploaded file,photo in public folder
jo ki hamare project mai ho rha h....public folder main.photo uploading ho rha h